README.adoc

```diff
@@ -45,94 +139,139 @@ If you request that URL through your web browser or curl, you'll receive a JSON
+To directly bind your data to your custom types, you need to specify the
+variable name to be exactly the same as the key in the JSON Document returned from the API.
+In case your variable name and key in JSON doc do not match, you can use `@JsonProperty`
+annotation to specify the exact key of the JSON document. (This example matches each
+variable name to a JSON key, so do not need that annotation here.)
 
-An additional class is needed to embed the inner quotation itself.
+We also need an additional class, to embed the inner quotation itself. The `Value` class
+fills that need and is shown in the following listing:
 
-`src/main/java/hello/Value.java`
+`src/main/java/com/example/consumingrest/Value.java`
 [source,java]
 ----
-include::complete/src/main/java/hello/Value.java[]
+include::complete/src/main/java/com/example/consumingrest/Value.java[]
 ----
 
-This uses the same annotations but simply maps onto other data fields.
-
-== Make the application executable
+This uses the same annotations but maps onto other data fields.
 
-Although it is possible to package this service as a traditional link:/understanding/WAR[WAR] file for deployment to an external application server, the simpler approach demonstrated below creates a standalone application. You package everything in a single, executable JAR file, driven by a good old Java `main()` method. Along the way, you use Spring's support for embedding the link:/understanding/Tomcat[Tomcat] servlet container as the HTTP runtime, instead of deploying to an external instance.
+== Finishing the Application
 
-Now you can write the `Application` class that uses `RestTemplate` to fetch the data from our Spring Boot quotation service.
+The Initalizr creates a class with a `main()` method. The following listing shows the
+class the Initializr creates:
 
-`src/main/java/hello/Application.java`
-[source,java,indent=0]
+====
+[src,Java]
 ----
-package hello;
+package com.example.consumingrest;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.web.client.RestTemplate;
-
-public class Application {
 
-    private static final Logger log = LoggerFactory.getLogger(Application.class);
+@SpringBootApplication
+public class ConsumingRestApplication {
 
-    public static void main(String args[]) {
-        RestTemplate restTemplate = new RestTemplate();
-        Quote quote = restTemplate.getForObject("https://gturnquist-quoters.cfapps.io/api/random", Quote.class);
-        log.info(quote.toString());
-    }
+	public static void main(String[] args) {
+		SpringApplication.run(ConsumingRestApplication.class, args);
+	}
 
 }
 ----
+====
```

Comment:
Why not do an `include` directive to fetch the class. You can tag (with `code`) just the chunk you need and apply `tag=code`.